integration_tests/tests/gamelift.rs

```diff
@@ -0,0 +1,17 @@
+#![cfg(feature = "gamelift")]
+
+extern crate rusoto_core;
+extern crate rusoto_gamelift;
+
+use rusoto_gamelift::{GameLift, GameLiftClient, ListFleetsInput};
+use rusoto_core::{DefaultCredentialsProvider, Region, default_tls_client};
+
+#[test]
+fn should_list_fleets() {
```

Comment:
Glad to see this works.  My first stab at it last year revealed a duplicate in the botocore definiton that had to get fixed upstream - https://github.com/boto/botocore/issues/954